build.gradle

```diff
@@ -35,7 +35,7 @@ subprojects {
     ext {
         // Freezing this to 1.X until https://github.com/spring-projects/spring-boot/issues/12649 is resolved
         slf4jVersion = "1.7.36"
-        protobufVersion = "3.3.0"
+        protobufVersion = "3.21.7"
```

Comment:
I guess it's better to not mix dependency changes and functional changes, if avoidable?

Reply:
This was to build Decaton on Apple silicon (that I'm using) on local but yeah, not necessary to include in this PR. will separate the PR